app/assets/javascripts/comments.coffee

```diff
@@ -1,11 +1,12 @@
 $ ->
   $('.create_comment').on 'click' , ->
     post_id = $('#post_id').val()
+    user_id = $('#author_id').val()
```

Comment:
@vakyl9r and @AlexSukov  I remind about commit message, it should be `#number: message` don't forget `:` and don't write `#12.1`